Standards/scs-0005-v1-project-governance.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Sovereign Cloud Standards
+type: Procedural
+status: Draft
+track: Global
+description: |
+  SCS-0005 outlines the structure and governance of the SCS community by the SCS Project Board and how this is elected.
+---
+
+## Introduction
+
+The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
+for a range of cloud infrastructure types as well as a modular open-source
+reference implementation.
+The project is governed by the _SCS Project Board_.
+
+## Role of the _SCS Project Board_
+
+The role of the _SCS Project Board_ is the overall governance of the SCS Community and Project.
+This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
+underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
+The _SCS Project Board_ itself is elected by the _SCS Community_.
+
+### Definitions
+
+#### _SCS Project_
+
+The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.
+
+#### _SCS Community_
+
+The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.
+
+### _SCS GitHub Organization_
+
+The _SCS GitHub Organization_ is this: [https://github.com/sovereigncloudstack](https://github.com/sovereigncloudstack)
+
+## Joining the SCS GitHub Organization
+
+Since being part of the GitHub organization comes with a set of responsibilities, joining the SCS GitHub Organization can be done by:
+
+- being invited by the _SCS Project Board_
+- submitting a request to be onboarded as a member to the _SCS Project Board_
+- have to existing members of the GitHub organization that nominate you
+
+One of these items is sufficient.
```

Comment:
```suggestion
One of these items is sufficient.

Actively contributing to one or several of the projects under the governance of the SCS project board should typically result in a membership. Please be aware of our [Code of Conduct](https://github.com/SovereignCloudStack/.github/blob/main/CODE_OF_CONDUCT.md).
```

Reply:
I want to express that we github org membership is a result of the contribution, not a consequence of happening to know an existing member. The latter is the mechanism, not the selection criteria.
This is up for discussion of course, so I won't just push this myself.

Reply:
I think this is a good addition. 👍